### DISCUSSION TITLE
Allure plugin for Azure

### DISCUSSION THREAD

**Discussion by anushri** [allure-framework/allure2#2305] [Questions & Support] [answered]
Hi 
Could you please share insight on how view/ generate reports on Azure our framework in RestAssured tech based.

Do we need this 
https://marketplace.visualstudio.com/items?itemName=MichaelClay.azure-pipelines-allure-report-viewer

Thanks

**Comment by baev** [comment, marked as answer]
Official Azure extension is now available https://marketplace.visualstudio.com/items?itemName=qameta.allure-azure-pipelines
